src/Xamarin.Android.Tools.Bytecode/Tests/Xamarin.Android.Tools.Bytecode-Tests.csproj

```diff
@@ -56,7 +56,13 @@
     <Compile Include="ParameterFixupTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <TestJar Include="java\**\*.java" />
+    <TestJar
+        Include="java\**\*.java"
+        Exclude="java\java\util\Collection.java"
```

Comment:
It's excluded/missing parameters so that the `Xamarin.Android.Tools.BytecodeTests.ParameterFixupTests.XmlDeclaration_FixedUpFromApiXmlDocumentation` test continues to work.

That test works by "fixing up" parameter names based on imported XML, but the import only works if the parameter names are "generated" (e.g. `p1`...). If `Collection.java` is compiled with `javac -parameters`, it will have *correct* parameter names, not generated names, which "breaks" the `ParameterFixupTests.XmlDeclaration_FixedUpFromApiXmlDocumentation()` tests.

This breakage might be acceptable -- i.e. we fixup the expected XML, and the test passes -- but by doing so, we're no longer testing/ensuring that parameter fixup *continues to work*.